riscv_config/warl.py

```diff
@@ -77,9 +79,15 @@ def islegal(self, value, dependency_vals=[]):
                 dep_search = self.exp.search(dep_str)
                 if csr_search is None or dep_search  is None:
                     logger.error('Warl legal string is wrong :\n\t' + legal_str)
-                    raise SystemExit
+                    raise SystemExit(1)
                 dep_csr = dep_search.group('csr')
                 dep_ind = dep_search.group('csr_ind')
+                if ':' in dep_ind:
+                    msb = int(dep_ind.split(':')[0],0)
```

Comment:
Should probably also check if msb >= lsb after this so that the indexing is correct.

Reply:
fixed